app/components/DeleteButton/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+*
+* DeleteButton
+*
+*/
+
+import React from 'react';
+
+import styles from './styles.css';
+
+function DeleteButton(props) {
+  return (
+    <button
+      className={styles.deleteButton}
+      onClick={() => {
+        const response = window.confirm(props.confirmText); // eslint-disable-line no-alert
+        if (response) {
+          props.onClick();
```

Comment:
Is the point here that there's a regular browser OK/Cancel popup for the deletion, and one needs to confirm?